src/cclib/writer/xyzwriter.py

```diff
@@ -36,7 +36,6 @@ def __init__(self, ccdata, splitfiles=False,
         self.do_lastgeom = lastgeom
         self.do_allgeom = allgeom
 
-        self.generate_repr()
```

Comment:
I only have one question before Karol or I merge; if you remove this method call from the writers, how do I use them from Python code?